lib/ansible/modules/network/aos/aos_ip_pool.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/python
+#
+# (c) 2017 Apstra Inc, <community@apstra.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: aos_ip_pool
+author: Damien Garros (@dgarros)
+version_added: "2.3"
+short_description: Manage AOS IP Pool
+description:
+  - Apstra AOS Ip Pool module let you manage your IP Pool easily. You can create
+    create and delete IP Pool by Name, ID or by using a JSON File. This module
+    is idempotent and support the I(check) mode. It's using the AOS REST API
+requirements:
+  - aos-pyez
```

Comment:
I'm not sure how old aos-pyez 0.6.0 is, though you may wish to detail that here.
`- "aos-pyez >= 0.6.0"`